tls/Makefile

```diff
@@ -19,7 +19,7 @@ SUB_BUILDS = $(patsubst %/Makefile, %, $(wildcard */Makefile))
 
 BITCODE_DIR?=../tests/saw/bitcode/
 
-BCS_1=s2n_handshake_io.bc s2n_handshake_type.bc s2n_connection.bc s2n_kex.bc s2n_tls13.bc s2n_quic_support.bc
+BCS_1=s2n_handshake_io.bc s2n_handshake_type.bc s2n_connection.bc s2n_kex.bc s2n_tls13.bc s2n_quic_support.bc extensions/s2n_extension_type.bc
```

Comment:
Does it work with just `s2n_extension_type.bc` (no extensions directory prefix)?

Reply:
Unfortunately no, but I've found a way of getting around this problem.